src/bastion/azext_bastion/custom.py

```diff
@@ -231,6 +231,22 @@ def _get_rdp_path(rdp_command="mstsc"):
     return rdp_path
 
 
+def _generate_rdp_file(port):
+    import os
+
+    rdp_file_content = (
+        f"full address:s:localhost:{port}\n"
+        f"alternate full address:s:localhost:{port}\n"
+        "use multimon:i:1\n"
```

Comment:
can you make this i:0 instead? I remember some concerns on not having multiple monitors and just a single one.